src/ck-core/ckreduction.C

```diff
@@ -2050,8 +2023,6 @@ CkNodeReductionMgr::CkNodeReductionMgr()//Constructor
 {
 #ifdef BINOMIAL_TREE
   init_BinomialTree();
-#elif CMK_BIGSIM_CHARM
-  init_BinaryTree();
```

Comment:
Is there no case where we'd want to use a binary tree? And if not, the BinaryTree code can probably be removed as well.